Clean-code-cookbook-guide.html

```
@@ -0,0 +1,1131 @@
+    }
+    .sidebar.open{ transform: translateX(0); }
+    .main{ margin-left: 0; padding: 70px 20px 80px; }
+    .nav-toggle{ display:block; }
+    .hero{ padding: 28px 22px; }
+    .hero h1{ font-size: 1.5rem; }
+  }
+</style>
+</head>
+<body>
+
+<button class="nav-toggle" id="navToggle" aria-label="メニューを開く" aria-controls="sidebar" aria-expanded="false"><i class="ti ti-menu-2"></i></button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)